docs/sphinx/src/pip-delete-this-directory.txt

```diff
@@ -0,0 +1,5 @@
+This file is placed here by pip to indicate the source was put
+here by pip.
+
+Once this package is successfully installed this source code will be
```

Comment:
Does this file need to be here? Is it related to the theme?

Reply:
No this was a file generated during the build. Will remove